### ISSUE TITLE
Lock down version of source-map

### ISSUE THREAD

**Issue by arosequist** [SitePen/remap-istanbul#160]
Currently, this uses `>=0.5.6` for the source-map package, which seems too broad.

They just pushed version 0.7.0, which [includes](https://github.com/mozilla/source-map/blob/86d105bfda30fc71aba95c21f2e665a384b5b69f/CHANGELOG.md) four breaking changes. The first ("new SourceMapConsumer now returns a Promise object") is breaking remap-istanbul.

It seems like remap-istanbul should require `0.5.6` or `~0.5.6`

**Comment by Dufgui** [comment]
according to https://github.com/marcules/karma-remap-istanbul/blob/master/package.json
It would be nice to have a 0.9.6 release. If not please notify karma bridge. 
Many thanks

**Comment by dylans** [comment]
@Dufgui 0.9.6 with just the change in the source-map dependency?

**Comment by Dufgui** [comment]
Yes actually all your 0.9 release cannot work

**Comment by paulblyth** [comment]
I'm also running into this problem (I think)

```
22 01 2018 09:43:14.735:ERROR [karma]: TypeError: sourceMap.originalPositionFor is not a function
at getMapping (/project/node_modules/remap-istanbul/lib/getMapping.js:27:26)
```

**Comment by dylans** [comment]
Ok, this should be resolved, and 0.10.1 and 0.9.6 have been released and published to npm. Please let me know if I did not get this right. Thanks!
